.github/workflows/ci.yml

```diff
@@ -90,10 +90,10 @@ jobs:
       - run: sudo apt update && sudo apt install -y libhdf5-dev
       - name: Api tests
         shell: bash -l {0}
-        run: python -m pytest modin/pandas/test/test_api.py
+        run: python -m pytest -n 2 modin/pandas/test/test_api.py
```

Comment:
Test changes should be in a separate PR.

Reply:
Ok I will remove this commit because it didn't help anyways. Windows tests always fail in parallel because of Ray bug https://github.com/ray-project/ray/issues/13213